fe/lib/components/JobCategorySuggest/JobCategorySuggestChoices.tsx

```diff
@@ -96,6 +101,24 @@ const JobCategorySuggestChoices = forwardRef<HTMLInputElement, Props>(
             );
           })}
         </RadioGroup>
+        {selectedJobCategory === 'Other' && (
+          <JobCategorySelect
+            client={client}
+            name={name}
+            id="jobCategoryId"
+            onSelect={(jobCategory: JobCategoryAttributesFragment) => {
+              const jobCategorySuggest = choices.find(
+                (choice) => choice.jobCategory.id.value === jobCategory.id.value,
+              );
```

Comment:
The `JobCategorySelect` component lets you select _any_ category, while `choices` is a subset that only contains a few top suggestions based on user input. You need to propagate the `jobCategory` parameter by itself to `onSelect`.